Description: Use either byte order when detecting compressed XML in test output
Author: Ben Burton <bab@debian.org>
Origin: upstream
Applied-Upstream: 43b5bbf400
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/utils/testsuite/runtest.sh
+++ b/utils/testsuite/runtest.sh
@@ -298,7 +298,8 @@
     if [ -e "$testout" ]; then
         echo "--------------------"
         magic="`hexdump -n2 -e '/2 "%x\n"' "$testout"`"
-        if [ "$magic" = 8b1f ]; then
+        # On s390, the two bytes are output in reverse order.  Try both ways.
+        if [ "$magic" = 8b1f -o "$magic" = 1f8b ]; then
             echo "Output (compressed):"
             if [ -z "$filter" ]; then
                 cat "$testout" | gunzip
